CMakeLists.txt

```diff
@@ -2,6 +2,10 @@
 #
 cmake_minimum_required(VERSION 3.15.0)
 
+# macOS 10.15 is required for certain C++17 features.
+# (This variable needs to be set before calling `project`.)
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15" CACHE STRING "Minimum OS X deployment version")
+
```

Comment:
Let's only set this externally in deployment logic (e.g., pip wheels scripts, conda scripts and yggdrasil).
By default, this is set to the system and we support 11.0+ (just dropped 10.15+, but it still works on Catalina as of today).
```suggestion
```

Reply:
Removed.